infra-templates/k8s/20/docker-compose.yml

```diff
@@ -0,0 +1,230 @@
+kubelet:
+    labels:
+        io.rancher.container.dns: "true"
+        io.rancher.container.create_agent: "true"
+        io.rancher.container.agent.role: environmentAdmin
+        io.rancher.scheduler.global: "true"
+        io.rancher.scheduler.affinity:host_label_ne: nopods=true
```

Comment:
The label for `proxy` and `kubelet` were previously a hard affinity. Should these also be configurable or remain hard always?